satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -626,13 +626,23 @@ model certRecord (
 
 create certRecord ( )
 delete certRecord ( where certRecord.id = ? )
-update certRecord ( where certRecord.id = ? )
 
 read one (
 	select certRecord
 	where  certRecord.id = ?
 )
 
+read one (
+	select certRecord
+	where  certRecord.publickey = ?
```

Comment:
Order by the read one by update_at also

Reply:
updated

Reply:
this read is by a public key which will always be unique. But I did get your thought, when you read one key by ID, you wanted the latest public key for that nodeID. I updated the code GetPublicKey() to return you always the lastest of the public key(s) for the given node ID.